tsl/src/partialize_finalize.c

```diff
@@ -147,6 +148,86 @@ collation_oid_from_name(char *schema_name, char *collation_name)
 	namel = lappend(namel, makeString(collation_name));
 	return get_collation_oid(namel, false);
 }
+
+#if PG14_GE
+/*
+ * Record serialized partials errata here:
+ * ============================================================================
+ * PostgreSQL versions < 14
+ * F_NUMERIC_AVG_DESERIALIZE serialized partial length = X
+ * F_NUMERIC_DESERIALIZE serialized partial length = X + 10
+ *
+ * PostgreSQL versions >= 14
+ * F_NUMERIC_AVG_DESERIALIZE serialized partial length = X + 16
+ * F_NUMERIC_DESERIALIZE serialized partial length = X + 10 + 16
```

Comment:
This looks confusing, maybe something like "In PostgreSQL >= 14 the serialized length of some numeric aggregates is 16 bytes longer than in previous versions, because they now support infinity values"?

Reply:
Done.